packages/files/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import { Restroom } from "@restroom-mw/core";
+import { ObjectLiteral } from "@restroom-mw/core/src/types";
+import { Zencode } from "@restroom-mw/zencode";
+import axios from "axios";
+import { NextFunction, Request, Response } from "express";
+import fs from 'fs'
+import https from "https";
+import {
+  DOWNLOAD,
+} from "./actions";
+import yauzl from 'yauzl';
+import extract from 'extract-zip';
+
+const TEMP_FILE = '/tmp/temp-zip-file-downloaded-from-restroom';
```

Comment:
Please add a TODO to fix this (no hardcoded) with `fs.mkdtemp` or `os.tempdir` so it's compatible with all OSes

Reply:
This is a major issue if you run like a restroom with a application server like `pm2` that run more concurrent processes for one app, and if you have many processes writing in the same place it'll be sad 🤣